speaksfer/settings/staging.py

```diff
@@ -0,0 +1,5 @@
+from speaksfer.settings.base import ALLOWED_HOSTS
+
+DEBUG = True
+
+ALLOWED_HOSTS += ["localhost", "127.0.0.1"]
```

Comment:
we do not yet have a staging server so you can remove ALLOWED_HOSTS here

Reply:
Done